### PR TITLE
Implement arm timesave functions into BlueBackPixel.java

### PR DESCRIPTION
**Pull request by gwarby**

### CODE REVIEW

File: BlueBackPixel.java [hunk 1]
```diff
@@ -174,106 +174,92 @@ public void onError(int errorCode)
...
+                                    // ...but also don't go back to far into frames
+        drive(0, 0, -72, DRIVE_POWER);      // CCW 83 to face backdrop
+        sleep(50);
+        armraise(75, 0.3);         // raise arm 120 deg (all the way back/up for placing pixel on board)
+        drive(24, 0, 0, DRIVE_POWER);       // FWD 25.5" toward backdrop
+        drive(0, 0, 180, DRIVE_POWER);      // CW 180 deg to back into backdrop
+        reverseFlipper();           // put flipper in rev pos for placing pixel on board
+        drive(0, -9, 0, DRIVE_POWER);       // left 10" along backdrop
+        armraisewait(45, 0.127);        // slow down to avoid tipping over
+        armextend(-4,0.31);         // drop pixel from lowest pos to avoid bouncing as much as possible
```
Comment (lcdnbl) [inline BlueBackPixel.java]: @gwarby I think this BlueBack middle section now tries to retract the arm 4 inches twice.  (here on line 250 & also on line 240)

Comment (gwarby) [reply BlueBackPixel.java]: I removed the one on line 250, leaving just the retraction on 239.
Feel free to merge and close the PR if everything else looks good.

Comment (gwarby) [reply BlueBackPixel.java]: Actually, wait until new auto is merged, otherwise I'm not sure it will merge properly

**Comment by gwarby** [comment]
Moved updated code over to front-autonomous branch

**Pull request closed without merging**